benchmarks/TFLite/android-adreno.cmake

```diff
@@ -27,7 +27,8 @@ iree_benchmark_suite(
     "android-adreno"
 
   MODULES
-    "${DEEPLABV3_FP32_MODULE}"
+    # TODO(#10748): Disabled since the iree-import-tflite fails on this model.
```

Comment:
Also need to clean up https://github.com/iree-org/iree/blob/main/benchmarks/generated_benchmark_suites.cmake

Reply:
@hcindyl what does clean up mean?

Reply:
See the failure at https://github.com/iree-org/iree/actions/runs/3236387277/jobs/5302370270. The benchmark build script still tries to render deeplabv3 because it is coded in generated_benchmark_suites.cmake. We need to remove the reference in that file.

Reply:
I think the way is to re-run `./build_tools/benchmarks/generate_cmake_benchmark_suites.py`

Reply:
ugh, to re-run the script we will need to clean up all the references in `build_tools/python/benchmark_suites`... :( Maybe just commenting out the generated cmake file is easier...

Reply:
I have no idea of the cmake recipes here.... If someone has a better understanding, could you help disable the benchmark. Pull this branch (bump-llvm-20221007) make change, push to branch?
@pzread maybe?

Reply:
`IREE_BUILD_EXPERIMENTAL_PYTHON_GENERATED_BENCHMARKS` is not enabled so `generated_benchmark_suites.cmake` shouldn't be the problem. It failed at `benchmark_suites/TFLite/deeplabv3.tflite.mlir` which is the old benchmark suite. I'm checking what's going on

Reply:
ahh... the PR missed here <https://github.com/iree-org/iree/blob/744668647eb8a4b25ef559a34175d30db3fd2451/benchmarks/TFLite/CMakeLists.txt#L19-L31>

Reply:
You also need to disable deeplab for mali
https://github.com/iree-org/iree/blob/caa7842316463ec55d34df153d8450a88bd05937/benchmarks/TFLite/android-mali.cmake#L30

To make debug easier, you can also common out this section
https://github.com/iree-org/iree/blob/caa7842316463ec55d34df153d8450a88bd05937/benchmarks/TFLite/CMakeLists.txt#L19

Reply:
I'll push the change